OracleDatabase/SingleInstance/extensions/prebuiltdb/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for pre-built database for gaining speedup in CI/CD pipelines
+#
+# REQUIREMETNS FOR THIS IMAGE
+# ----------------------------------
+# Any release of prebuilt oracle/database base docker image
```

Comment:
Probably use 'container' instead of 'docker'

Reply:
Resolved.